magento/pwa-studio
# Checkout page region field fix.

## Description

The shipping information form in the checkout page has a bug where it does not pick up the region code provided in the cart page if the user does not have a default address assigned to their account. This PR addresses that issue.

## Related Issue
Closes PWA-671

### Verification Stakeholders
@dpatil-magento 

### Verification Steps
1. Create an account and get to the cart page after adding an item to the cart.
2. Estimate the shipping costs by giving the country, state, and zip code information.
3. Get to the checkout page, see if the region code is being carried over.
4. Perform the above steps on a country that does not support regions as well. For instance, United Kingdom.

## Screenshots / Screen Captures (if appropriate)

United States

![image](https://user-images.githubusercontent.com/35203638/111329576-31b4ed80-863d-11eb-9ec5-16866c98c009.png)

United Kingdom

![image](https://user-images.githubusercontent.com/35203638/111329289-03371280-863d-11eb-8219-2647a60dd0dc.png)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
